torch/_torch_docs.py

```diff
@@ -1076,28 +1076,27 @@ def merge_dicts(*dicts):
     >>> c = torch.asarray(a, copy=True)
     >>> a.data_ptr() == c.data_ptr()
     False
-
-    >>> a = torch.tensor([1, 2, 3], requires_grad=True).float()
```

Comment:
Originally this line would fail with the following error so I moved `float` inside tensor creation.
```
Failed example:
    a = torch.tensor([1, 2, 3], requires_grad=True).float()
Exception raised:
    Traceback (most recent call last):
      File "/home/oscar/miniconda3/envs/pytorch-deps/lib/python3.9/doctest.py", line 1336, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest torch.asarray[6]>", line 1, in <module>
        a = torch.tensor([1, 2, 3], requires_grad=True).float()
    RuntimeError: Only Tensors of floating point and complex dtype can require gradients
```